docs/modules/reference/pages/glossary.adoc

```diff
@@ -109,31 +115,34 @@ It provides visualization capabilities for data indexed in an Elasticsearch clus
 xref:operation:admin/webui/opsboard/dashlet/ksc.adoc#ksc[KSC reports]:: Key SNMP customized (KSC) reports provide a way to generate prefabricated graphical views of collected data.
 They let you display data from different devices and sources (SNMP, ICMP, HTTP) on one page.
 
-Kubernetes:: An open-source container orchestration system for automating software deployments, scaling, and management.
+https://kubernetes.io/[Kubernetes]:: An open-source container orchestration system for automating software deployments, scaling, and management.
 Originally designed by Google, the Cloud Native Computing Foundation now maintains it.
-(See https://kubernetes.io/[Kubernetes].)
 
 Location:: Also referred to as "monitoring location" or "Minion location".
 Nodes assigned to a monitoring location will be monitored by Minions assigned to the same location.
 This is distinct from geolocation, which is the actual physical location of the node.
 
+Management Information Base (MIB):: A database used for managing entities on a network.
+Used in {page-component-title} to facilitate Transparent Bridge Discovery.
+(See xref:operation:topology/enlinkd/layer-2/bridge-discovery.adoc[Bridge Discovery].)
+
 https://www.opennms.com/meridian/[Meridian]:: A subscription-based, optimized, and stable version of the OpenNMS Horizon platform that includes only the most stable and secure Horizon features.
 It is distributed via annual release with monthly security patches.
 
 Message broker:: When using Minions and Sentinels, a message broker is required for communication between servers.
 {page-component-title} has an embedded ActiveMQ server available for smaller deployments.
 You can swap the broker with a dedicated ActiveMQ, gRPC, or Kafaka cluster to provide scalability and load balancing.
 
-xref:development:minion/introduction.adoc#minion[Minion]:: Instance of the Karaf OSGi service that enables OpenNMS to monitor devices and services in locations that OpenNMS cannot reach.
+xref:development:minion/introduction.adoc#minion[Minion]:: An instance of the Karaf OSGi service that enables OpenNMS to monitor devices and services in locations that OpenNMS cannot reach.
```

Comment:
Many operator users don't know what a "Karaf OSGi service" is, and don't really need to know unless they are writing code.
```suggestion
xref:development:minion/introduction.adoc#minion[Minion]:: A distributed component that enables OpenNMS to monitor devices and services in locations that OpenNMS cannot reach.
```

Reply:
We should update the cloud docs to reflect this.